dashboard/legacy/middleware/channels_api.rb

```diff
@@ -264,19 +264,6 @@ class ChannelsApi < Sinatra::Base
     {skip_content_moderation: value}.to_json
   end
 
-  #
-  # GET /v3/channels/<channel-id>/privacy-profanity
-  #
-  # Get an indication of privacy/profanity violation.
-  #
-  get %r{/v3/channels/([^/]+)/privacy-profanity} do |id|
```

Comment:
Removing this endpoint since we removed the only use in https://github.com/code-dot-org/code-dot-org/pull/65397/files.